qa-tests-backend/src/test/resources/logback-test.xml

```diff
@@ -2,7 +2,7 @@
     <withJansi>true</withJansi>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%d{HH:mm:ss} | %-5level | %-25logger{0} | %m%n%rEx{full,
+            <pattern>%replace(%replace(%thread){"^Test worker", ""}){".*worker-(\d+)", "thread-$1 | "}%d{HH:mm:ss} | %-5level | %-25logger{0} | %m%n%rEx{full,
```

Comment:
example log line: 
`Time-limited test03:42:18 | INFO  | PolicyConfigurationTest   | Starting testcase`
I think we may want to add ` | ` before date and maybe a field size like we do for logger?

Reply:
Fixed.
```
RuntimePolicyTest STANDARD_OUT
    22:54:58 | INFO  | RuntimePolicyTest         | Starting testsuite
    22:54:58 | DEBUG | Kubernetes                | Namespace qa already exists
    22:54:59 | DEBUG | Kubernetes                | Told the orchestrator to createOrReplace runtimenginx
```